notifier/user.py

```diff
@@ -3,31 +3,54 @@
 """
 import logging
 import sys
-
-from django.conf import settings
 import requests
 import six
+import datetime
+
+from django.conf import settings
+from django.core.cache import cache
+from edx_rest_api_client.client import get_oauth_access_token
 
 
 logger = logging.getLogger(__name__)
 
 
 DIGEST_NOTIFICATION_PREFERENCE_KEY = 'notification_pref'
 LANGUAGE_PREFERENCE_KEY = 'pref-lang'
+AUTH_TOKEN_CACHE_KEY = 'access_token_for_notifier'
 
 
 class UserServiceException(Exception):
     pass
 
+
+def get_access_token():
+    token = cache.get(AUTH_TOKEN_CACHE_KEY)
+    if token is None:
+        oauth_url = '{root}/access_token'.format(root=settings.LMS_URL_BASE)
```

Comment:
I believe it should be `'{root}/oauth2/access_token'.format(root=settings.LMS_URL_BASE)`